### DISCUSSION TITLE
typings in v5.75.0 cause errors in plugin TSC runs

### DISCUSSION THREAD

**Discussion by localyost3000** [webpack/webpack#16639] [Q&A]
# Bug report
**What is the current behavior?**
running tsc on a webpack plugin throws hundreds of errors on the `node_modules/webpack/types.d.ts` file

**If the current behavior is a bug, please provide the steps to reproduce.**
1. install webpack@latest
2. use webpack library in ts file
3. run tsc

I encountered this issue when trying to push a new version of my [webpack-nomodule-plugin](https://github.com/swimmadude66/webpack-nomodule-plugin)

https://github.com/swimmadude66/webpack-nomodule-plugin/actions/runs/3818823788/jobs/6495964117

**What is the expected behavior?**
no errors thrown, tsc proceeds as it does in v5.74.0 with the same configuration

**Other relevant information:**
webpack version:  5.75.0
Node.js version: 10+ (tested with v10.24.1) and v16
Operating System: Linux (GHA runner)
Additional tools: GithubActions

**Comment by alexander-akait** [comment]
Hello, can you provide your tsc version?

**Reply by localyost3000** [reply]
8.19.2
